pkg/syncer/syncers/downsyncer.go

```diff
@@ -0,0 +1,181 @@
+			return err
+		}
+	} else {
+		if downstreamResource != nil {
+			// update
+			ds.logger.V(3).Info(fmt.Sprintf("  update '%s' in upstream since it's found", resourceToString(resourceForDown)))
+			upstreamResource.SetResourceVersion("")
+			upstreamResource.SetUID("")
+			upstreamResource.SetManagedFields(nil)
+			applyConversion(upstreamResource, resourceForDown)
+			if _, err := downstreamClient.Update(resourceForDown, upstreamResource); err != nil {
+				ds.logger.Error(err, fmt.Sprintf("failed to update resource on downstream '%s'", resourceToString(resourceForDown)))
+				return err
+			}
+		} else {
+			msg := fmt.Sprintf("downstream resource is nil even if there is no error '%s'", resourceToString(resourceForDown))
+			return errors.New(msg)
+		}
+	}
+	return nil
+}
+
+func (ds *DownSyncer) BackStatusOne(resource edgev1alpha1.EdgeSyncConfigResource, conversions []edgev1alpha1.EdgeSynConversion) error {
+	upstreamClient, downstreamClient, err := ds.getClients(resource, conversions)
+	if err != nil {
+		ds.logger.Error(err, fmt.Sprintf("failed to get client '%s'", resourceToString(resource)))
+		return err
+	}
+	resourceForDown := convertToDownstream(resource, conversions)
+	downstreamResource, err := downstreamClient.Get(resourceForDown)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			ds.logger.V(3).Info(fmt.Sprintf("  not found '%s' in downstream", resourceToString(resourceForDown)))
+			ds.logger.V(3).Info(fmt.Sprintf("  skip status upsync '%s'", resourceToString(resourceForDown)))
+			return nil
+		} else {
+			ds.logger.Error(err, fmt.Sprintf("failed to get resource from upstream '%s'", resourceToString(resourceForDown)))
+			return err
+		}
+	}
+	status, found, err := unstructured.NestedMap(downstreamResource.Object, "status")
+	if err != nil {
+		ds.logger.Error(err, fmt.Sprintf("failed to extract status from downstream object '%s'", resourceToString(resourceForDown)))
+		return err
+	} else if !found {
+		ds.logger.Info(fmt.Sprintf("no status field downstream object '%s'", resourceToString(resourceForDown)))
+		return nil
+	}
+	resourceForUp := convertToUpstream(resource, conversions)
+	upstreamResource, err := upstreamClient.Get(resourceForUp)
+	if err != nil {
+		if k8serrors.IsNotFound(err) {
+			ds.logger.V(3).Info(fmt.Sprintf("  not found '%s' in upstream", resourceToString(resourceForUp)))
+			ds.logger.V(3).Info(fmt.Sprintf("  skip status upsync '%s'", resourceToString(resourceForUp)))
+			return nil
+		} else {
+			ds.logger.Error(err, fmt.Sprintf("failed to get resource from upstream '%s'", resourceToString(resourceForUp)))
+			return err
+		}
+	}
+	_, found, err = unstructured.NestedMap(upstreamResource.Object, "status")
+	if err != nil {
+		ds.logger.Error(err, fmt.Sprintf("failed to extract status from upstream object '%s'", resourceToString(resourceForUp)))
+		return err
+	}
+	upstreamResource.Object["status"] = status
+	updatedResource := unstructured.Unstructured{
```

Comment:
Why is this copy being made?

Reply:
Maybe this has been left unintentionally. I confirmed that the status is properly upsynced without this copy. Thanks. Will be fixed in this PR.